modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/action/test/GetContentDashboardItemInfoMVCResourceCommandTest.java

```diff
@@ -75,16 +82,30 @@ public void setUp() throws Exception {
 
 	@Test
 	public void testServeResource() throws Exception {
-		FileEntry fileEntry = _addFileEntry();
+
+		AssetVocabulary assetVocabulary = AssetTestUtil.addVocabulary(
+			_group.getGroupId());
+
+		AssetCategory assetCategory1 = AssetTestUtil.addCategory(
+			_group.getGroupId(), assetVocabulary.getVocabularyId());
+		AssetCategory assetCategory2 = AssetTestUtil.addCategory(
+			_group.getGroupId(), assetVocabulary.getVocabularyId());
+
+		JournalArticle journalArticle = _addJournalArticle(
```

Comment:
Hey @beltranrengifo and @YurenaC !!!

It is not a good practice to couple our test (of the content dashboard api) to the journal entity

Reply:
or the file entity

Reply:
How do you suggest to create the content instead? 
The first thing that comes to my mind in this case, is create a DB script, like a set up that creates all the data needed, but it is not a common practice in the portal either.

Happy to hear any ideas :D

Reply:
We should follow the same approach that we do in com.liferay.analytics.reports.test.analytics.reports.info.item.MockObjectAnalyticsReportsInfoItem. So we create an entity for the tests.

So we could decouple the API for any implementation. This way,  the changes in Journal won 't affect the ContentDashboardAPI

Reply:
You can see a full example of how we use it in com.liferay.analytics.reports.web.internal.item.action.provider.test.AnalyticsReportsContentDashboardItemActionProviderTest.